cli/parse.go

```diff
@@ -76,6 +77,18 @@ func parse(env environ.Values, r io.Reader) (*run.Config, error) {
 		return nil, err
 	}
 	cfg.Driver = d
+
+	// Add plugin lookup directories to $PATH. We add the directory named plugin
+	// found in the project root by default.
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	err = environ.AddDirsToPath(append(c.PluginDirs, filepath.Join(wd, "plugins")))
```

Comment:
I don't think we can just add these dirs to the PATH. We *only* want to run commands from these directories.  If we use the current path too, then someone could make a script that calls any executable on the user's machine.

I think we should use exec.Lookpath with the list of directories directly, so we're just manually looking for the first one we can run.  Then we're also not mucking with the PATH for the entire executable.